neofetch

```diff
@@ -3202,6 +3202,49 @@ END
             # Default fallback font hardcoded in terminal-preferences.c
             [[ -z "$term_font" ]] && term_font="Monospace 12"
         ;;
+
+        "conemu-"*)
+            local ce_arg_list
+            local ce_arg_idx
+            local ce_conf
+
+            # Could have used `eval set -- "$ConEmuArgs"` instead for arg parsing but eval is evil...
+            readarray -t ce_arg_list < <(xargs -n1 printf "%s\n" <<< "${ConEmuArgs-}")
+
+            for ce_arg_idx in "${!ce_arg_list[@]}"; do
+                # Search for "-LoadCfgFile" arg
+                [[ "${ce_arg_list[$ce_arg_idx]}" != -LoadCfgFile ]] && continue
+
+                # Conf path is the next arg
+                ((++ce_arg_idx))
+                ce_conf="${ce_arg_list[$ce_arg_idx]}"
+                break
+            done
+
+            # https://conemu.github.io/en/ConEmuXml.html#search-sequence
+            local ce_seq=(
```

Comment:
This could just be:

```sh
for ce_conf in str str str
               str str str; do

done
```

Reply:
Like this ?

```bash
for ce_conf in "$ce_conf" "$ConEmuDir\ConEmu.xml" "$ConEmuDir\.ConEmu.xml"
               "$ConEmuBaseDir\ConEmu.xml" "$ConEmuBaseDir\.ConEmu.xml"
               "$APPDATA\ConEmu.xml" "$APPDATA\.ConEmu.xml"; do

```

Reply:
Yeah, saves creating a local variable and accompanying array.